docs/theories/topological_spaces.md

```diff
@@ -0,0 +1,175 @@
+# Maths in lean : Topological Spaces.
+
+The `topological_space` typeclass is defined in mathlib,
+in `analysis/topology/topological_space.lean`. There are over 4500
```

Comment:
to give the lines of codes is quiet strange given that we are in the same repository...